packages/storybook-utils/src/index.ts

```diff
@@ -1,4 +1,6 @@
+export * from "@storybook/csf";
```

Comment:
Would prefer to import directly from storybook, e.g. `"storybook/internal/types";`

Reply:
done, thanks for the hint